.docker/Dockerfile.debian

```diff
@@ -19,7 +19,8 @@ RUN set -ex && \
         git \
         openssl \
         openssh-client \
-        ca-certificates && \
+        ca-certificates \
+        iputils-ping && \
```

Comment:
Good with me. Does it include ping6 ?

Reply:
https://packages.debian.org/sid/amd64/iputils-ping/filelist

Looks like it